section4/exercises/classMethods.js

```diff
@@ -28,13 +28,71 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
-
-  // ADD CODE
+changeProtein() {
+  this.protein = "steak"
```

Comment:
These methods could be called `changeProteinToSteak` or `addCilantroToToppings` - they are very specific. You can use parameters in these methods to make what you change or add more variable.